scripts/eslint/rules/no-double-space.ts

```diff
@@ -30,7 +30,7 @@ export = createRule({
         };
 
         const isRegexLiteral = (node: TSESTree.Node | null): boolean => {
-            return !!(node && node.type === AST_NODE_TYPES.Literal && node.regex);
+            return !!(node && node.type === AST_NODE_TYPES.Literal && Object.prototype.hasOwnProperty.call(node, "regex"));

```

Comment:
how about just use `in`.

Reply:
I think that in this case, we don't need to check the `regex` in the inheritance chain.